draft-ietf-quic-recovery.md

```diff
@@ -897,9 +897,28 @@ congestion window, which is recommended to be the minimum of
 max_datagram_size is the current maximum size of a datagram for the connection,
 not including UDP or IP overhead.
 
-As an example of a well-known and publicly available implementation of a flow
-pacer, implementers are referred to the Fair Queue packet scheduler (fq qdisc)
-in Linux (3.11 onwards).
+Endpoints can implement pacing as they choose. A perfectly paced sender spreads
```

Comment:
I would propose moving this paragraph up one, and adding this to the start of the previous paragraph: "Practical considerations, such as packetization, scheduling delays, and computational efficiency, can cause a sender to deviate from this rate over time periods that are much shorter than a round-trip time. Sending multiple packets into the network without any delay ..."

Reply:
Will do.  I also noticed that that paragraph refined the initial congestion window, so I replaced the (re)definition with a section reference.